### DISCUSSION TITLE
Suggestion: Make the Floating Islands accessible all year again.

### DISCUSSION THREAD

**Discussion by JohnSmith2007** [Monika-After-Story/MonikaModDev#8600] [QA - tech support + general]
I just hit 500 affection a few days ago, and i can't seem to find the Floating Islands topic. After searching around here in the Github, Apparently it's not accessible during the winter. Which, in my opinion, is really dumb.

**Comment by Booplicate** [comment]
You can access the islands any time, but you can't unlock (see for the first time) during winter, that's because they don't look as good.

**Reply by JohnSmith2007** [reply]
Damn, sad day.
